document/CoreFormat.mdk

```diff
@@ -678,26 +801,26 @@ A `result` object may contain a property named `toolFingerprint` whose value is
 that contributes to the process of uniquely identifying the result.
 [Annex A](#Annex-A) explains how a result management system can use this value.
 
-### `executionFlows` property { #result-executionFlows }
+### `codeFlows` property { #result-codeFlows }
 
-A `result` object may contain a property named `executionFlows` whose value is an array of arrays of
+A `result` object may contain a property named `codeFlows` whose value is an array of arrays of
 `annotatedCodeLocation` objects (&sect;[#annotatedCodeLocation]).
-For convenience, in this context we refer to an array of `annotatedCodeLocation` objects as an `executionFlow` object;
-hence we may say that the value of the `executionFlows` property is an array of `executionFlow` objects.
-The `executionFlows` property is intended for use by static analysis tools that perform flow analysis
+For convenience, in this context we refer to an array of `annotatedCodeLocation` objects as a `codeFlow` object;
+hence we may say that the value of the `codeFlows` property is an array of `codeFlow` objects.
+The `codeFlows` property is intended for use by static analysis tools that perform flow analysis
 in the process of producing results.
 
-Each `executionFlow` object in the `executionFlows` array shall describe a single execution flow.
-An execution flow is a sequence of locations visited by the tool in the course of performing a flow analysis.
-An execution flow is in turn represented by an array of `annotatedCodeLocation` objects.
+Each `codeFlow` object in the `codeFlows` array shall describe a single code flow.
+A code flow is a sequence of locations visited by the tool in the course of performing a flow analysis.
```

Comment:
A code flow is a sequence of locations that, taken together, help illustrate or otherwise drive evaluation of a possible defect.